packages/components/psammead-story-promo/src/index.stories.jsx

```diff
@@ -0,0 +1,45 @@
+import React, { Fragment } from 'react';
+import { storiesOf } from '@storybook/react';
+import { withKnobs, text } from '@storybook/addon-knobs';
+import { inputProvider } from '@bbc/psammead-storybook-helpers';
+import Img from '@bbc/psammead-image';
```

Comment:
Thought on this being `Image` so that when it's used lower down it is `<Image>` and not `<Img>` which could be confused with the real HTML `<img>`

Reply:
SGTM

Reply:
It makes sense 👍

Reply:
Done :)